feat: add a new DAG example to run DVT (#485)
GoogleCloudPlatform/professional-services-data-validator: docs/installation.md

If native installation is not an option for you, you can create a Docker image for this tool.  

Here's an [example](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/samples/docker/README.md) on how you can create a sample docker image for this tool.

## Automate using Apache Airflow

> It is an option to orchestrate the validation and implement it as a task within an Airflow DAG.

Reviewer: ```suggestion
You can orchestrate DVT by running a validation as a task within an Airflow DAG.
```

> It is an option to orchestrate the validation and implement it as a task within an Airflow DAG.

Reviewer: ```suggestion
You can orchestrate DVT by running a validation as a task within an Airflow DAG.
```
